insights/util/symbolic_name.py

```diff
@@ -0,0 +1,26 @@
+"""
+A method to get the spec name of some special symbolic_name in the uploader.json
+"""
+
+SPECIAL_SYMBOLIC_NAME_IN_UPLODER_JSON = {
+    'getconf_pagesize': 'getconf_page_size',
+    'krb5_conf_d': 'krb5',
+    'limits_d': 'limits_conf',
+    'machine_id1': 'machine_id',
+    'machine_id2': 'machine_id',
+    'machine_id3': 'machine_id',
+    'modprobe_conf': 'modprobe',
+    'modprobe_d': 'modprobe',
+    'netstat__agn': 'netstat_agn',
+    'ps_auxwww': None,
```

Comment:
This `ps_auxwww` was not in `DefaultSpecs`, but `ps_auxww` did. Should this also be mapped?